Formula/ecl.rb

```diff
@@ -32,20 +32,25 @@ class Ecl < Formula
   depends_on "texinfo" => :build # Apple's is too old
   depends_on "bdw-gc"
   depends_on "gmp"
-  depends_on "libffi"
+  uses_from_macos "libffi", since: :catalina
 
   def install
     ENV.deparallelize
 
     # Avoid -flat_namespace usage on macOS
     inreplace "src/configure", "-flat_namespace -undefined suppress ", "" if OS.mac?
 
+    libffi_prefix = if (sdk = MacOS.sdk_path_if_needed)
+      sdk
```

Comment:
```suggestion
    libffi_prefix = if MacOS.version >= :catalina
      MacOS.sdk_path
```

We're only using libffi on >= Catalina while `MacOS.sdk_path_if_needed` is non-nil on >= Mojave.

Reply:
Fixed.